students/Sean_Tasaki/Lesson06/test_mailroom_part4.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import mailroom_part4 as mailroom 
+from collections import defaultdict
+import pytest
+
+@pytest.fixture()
+def resource_a():
+    s = 'Sean'
+    return s
```

Comment:
Generally fixtures would be used for more complex data than a single string and would be shared between multiple test cases